---
string mode
uint16 IntegrationTime
string HDR
uint16 HDRTime
